### PR TITLE
Connect search participants

### PR DESCRIPTION
**Pull request by nuzurie**
- Search for student
- Display results

**Search by name**
Screenshot 2022-03-24 at 10 53 28 PM

**Filter by class**
image

**Participant search with filter set**
image

### CODE REVIEW

File: src/screens/TeamFormation/Tabs/Participants.js [hunk 1]
```diff
@@ -1,19 +1,60 @@
...
     render() {
       return (
-        <ListContainer marginBottom={50}>
+        <ListContainer marginBottom={50} onSearch={this.onParticipantSearch}>
            {/* placing these for visualization */}
-          <ParticipantListItem
+           {this.state.participants ? this.state.participants.map(participant => (
+             <ParticipantListItem id={participant.id}
+             participantName={`${participant.first_name} ${participant.last_name}`}
+             // commonClasses can be shown only if we keep state of current user's classes
```
Comment (nuzurie) [inline src/screens/TeamFormation/Tabs/Participants.js:87]: Common classes we can show if needed. Will require us to store the currently logged user's classes in a global state and then perform operations on it

**Review by yasfahm — APPROVED** [review]
Looks great Zub!! 🎉

**Review by car12lin12 — COMMENTED** [review]
can you make the search field for participant name a separate text input from class field? it was my first time trying and I didn't realize I had to press different buttons to search by user name or course name, perhaps you can paste the class input field below the name input field? so then we can see the texts that say "search for participant" and "search by class", I think it will be clearer

**Review by car12lin12 — APPROVED** [review]
Great work Zubair :rocket:

**Review by car12lin12 — APPROVED** [review]
Works well Zubair!!

**Pull request merged**